### ISSUE TITLE
Pull Request for Arduino Support

### ISSUE THREAD

**Issue by pschatzmann** [mborgerding/kissfft#76]
I was wondering if you would accept a pull request to add Arduino Support which has the following changes
- new file library.properties in root.
- new file kiss_arduino_config.h in root where we can define FIXED_POINT. 
- the following change in kiss_fft.h
```
#if defined(ARDUINO) && !defined(IGNORE_ARDUINO_CONFIG)
#include "kiss_arduino_config.h"
#endif
```

**Comment by dizcza** [comment]
I saw your fork. Please rename IGNORE_ARDUINO_CONFIG to a more project-specific name like KISSFFT_IGNORE_ARDUINO_CONFIG.

**Comment by dizcza** [comment]
... and putting yourself as the author discredits Mark's invaluable work.

**Comment by pschatzmann** [comment]
Done, but I did not get any answer from [mborgerding](https://github.com/mborgerding), so I did not submit any pull request

**Comment by dizcza** [comment]
Yeah, the support seems to be abandoned, as I've also seen another PR here that I wanted to be merged.

Anyhow, I linked your library against a copy-pasted kissfft lib from [this](https://github.com/atomic14/esp32-tft-audio-monitor/tree/main/lib/kissfft) project and it worked out fine.

**Comment by pschatzmann** [comment]
Cool, my major goal was to be able to provide some alternative implementations in my [Arduino Audio Tools](AudioEspressifFFT).  You might also have a look at esp-dsp from Espressif...
